server/service/appconfig.go

```diff
@@ -109,7 +106,22 @@ func (svc *Service) AppConfig(ctx context.Context) (*fleet.AppConfig, error) {
 		}
 	}
 
-	return svc.ds.AppConfig(ctx)
+	ac, err := svc.ds.AppConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	if ac.SMTPSettings.SMTPPassword != "" {
+		ac.SMTPSettings.SMTPPassword = "********"
+	}
+
+	var jiraIntegrations []*fleet.JiraIntegration
```

Comment:
Is the extra slice necessary?

Reply:
Hm, no, you're right. We are copying from the cache.